jsonnet/components/prometheus-user-workload.libsonnet

```diff
@@ -291,6 +291,11 @@ function(params)
     },
 
     prometheus+: {
+      metadata+: {
+        annotations+: {
+          'operator.prometheus.io/controller-id': 'openshift-user-workload-monitoring/prometheus-operator',
```

Comment:
```suggestion
          'operator.prometheus.io/controller-id': 'openshift-monitoring/prometheus-operator',
```

Reply:
You mean jsonnet/components/alertmanager-user-workload.libsonnet? fix it in next push

Reply:
sorry wrong file, I meant in https://github.com/openshift/cluster-monitoring-operator/pull/2309/files#diff-f17439bae8b1428d504f85741b6874353853a4290de644751f8b0a02952da847

Reply:
done